doc/lua_api.txt

```diff
@@ -2521,12 +2521,17 @@ Styling Formspecs
 
 Formspec elements can be themed using the style elements:
 
-    style[<name>;<prop1>;<prop2>;...]
-    style_type[<type>;<prop1>;<prop2>;...]
+    `style[<name 1>,<name 2>,...;<prop1>;<prop2>;...]`
+    `style_type[<type 1>,<type 2>,...;<prop1>;<prop2>;...]`
 
 Where a prop is:
 
-    property_name=property_value
+    `property_name=property_value`
+
+A name/type can optionally be a comma separated list of names/types, like so:
+
+    `world_delete,world_create,world_configure`
+    `button,image_button`
```

Comment:
Adding backticks is not needed. Indented lines like these here will be rendered as code block.